### DISCUSSION TITLE
can't install expo

### DISCUSSION THREAD

**Discussion by viegandev** [community/community#21519] [Other Feature Feedback, Questions, & Ideas]
<pre><code>npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! libxmljs@0.19.7 install: `node-pre-gyp install --fallback-to-build --loglevel http`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the libxmljs@0.19.7 install script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR! C:\Users\antho\AppData\Roaming\npm-cache\_logs\2019-11-19T12_00_15_255Z-debug.log
</code></pre>
<p>windows 10</p>
<p>node.js 12.13.1</p>
<p>python 3.8</p>
<p>Hello, I’m trying to download expo then I typed “npm install -g expo-cli”</p>
<p>At the end I always get the answer above.</p>
<p>I uninstalled node.js and got it back, updated python.</p>
<p>Used PowerShell as an administrator instead of cmd</p>
<p>Tried “npm install --global --production windows-build-tools” after getting python updated</p>
<p>I’ve checked every single post related to this problem and nothing worked for me.</p>

**Comment by fire-eggs** [comment, marked as answer]
<p>Did you try their <a href="https://forums.expo.io/" rel="nofollow noopener">forums</a>?</p>
